releasetasks/test/test_final_verification.py

```diff
@@ -28,6 +29,7 @@ def setUp(self):
             updates_enabled=False,
             bouncer_enabled=False,
             push_to_candidates_enabled=False,
+            postrelease_version_bump_enabled=True,
```

Comment:
most tests focus on a single template. by setting postrelease_version_bump_enabled=True for final verify and others, it will force them to be dependent on version_bump.yml.tmpl. Do we want that? should we make all the other template conditionals (updates_enabled, bouncer_enabled, etc) True for each test?